### ISSUE TITLE
dropout for eval

### ISSUE THREAD

**Issue by elbaro** [titu1994/neural-image-assessment#29]
Hi, do you turn off Dropout when evaluating?
I couldn't find the related code.

**Comment by titu1994** [comment]
That is managed by Keras internally using the Keras backend function called set training mode to true or false. 

When training or testing a model for evaluation, Keras changes that flag internally.
